agent/tools/jin10.py

```
@@ -134,9 +146,23 @@ def _run(self, history, **kwargs):
         except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)